src/js_subscriptions.hpp

```diff
@@ -492,6 +492,16 @@ void SubscriptionSetClass<T>::wait_for_synchronization_impl(ContextType ctx, Obj
 
                 Function<T>::callback(protected_ctx, protected_callback, protected_this, {result});
             }
+            else {
+                auto error = Object::create_obj(
```

Comment:
I know this is how we do it elsewhere, but could we easily use an `Error` instance instead?

Reply:
https://github.com/realm/realm-js/pull/4296/commits/60807dbdda602de87abb28689dc2c2d682753358